### PR TITLE
Ekran 1

### PR DESCRIPTION
**Pull request by hub-bla**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "lint": "yarn lint:format && yarn lint:fix "
   },
   "dependencies": {
-    "@tanstack/react-query": "^4.19.1",
+    "@tanstack/react-query": "^4.20.9",
```
Comment (Safeheaded) [inline client/package.json:15]: Shouldn't there be @ant-design/icons in dependencies?

File: client/src/components/forms/FindBoxForm/FindBoxForm.module.less [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+.input,
+.select {
+  width: calc(@input-width - 200px);
+}
+
+.input {
+  position: relative;
+}
+
+// .input::before {
```
Comment (Safeheaded) [inline client/src/components/forms/FindBoxForm/FindBoxForm.module.less:29]: Is that comment needed?

File: client/src/components/forms/FormButton/FormButton.module.less [hunk 3]
```diff
@@ -1,9 +1,9 @@
-@import "less/variables.less";
+@import 'less/variables.less';
 
 .submitBtn {
   align-self: start;
   height: 2rem;
-  border-radius: 0.125rem;
+  border-radius: 0.125rem !important;
```
Comment (Safeheaded) [inline client/src/components/forms/FormButton/FormButton.module.less:6]: Something seems to be wrong with these border-radiuses - i will try to check it tomorrow

File: client/src/components/forms/FormError/index.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export * from './FormError';
```
Comment (Safeheaded) [inline client/src/components/forms/FormError/index.ts:1]: File not ending with new line

**Review by Safeheaded — APPROVED** [review]
Good job. Please, fix minor errors. As i've written in a comment - i'll look into these border radius thingy tomorrow (that `!important` is giving me a bad vibe).

**Review by Safeheaded — CHANGES_REQUESTED** [review]
Missclicked and approved by accident, sorry.

File: client/package.json (hunk 1 above)
Comment (hub-bla) [reply client/package.json:15]: Well, I don't see them on the master either

Comment (Safeheaded) [reply client/package.json:15]: Hmm, was it working for you right away or did you have to install the package?

File: client/src/components/forms/FormButton/FormButton.module.less (hunk 3 above)
Comment (Safeheaded) [reply client/src/components/forms/FormButton/FormButton.module.less:6]: Ok, seems to be the quickest way to override ant-design styles

File: client/package.json (hunk 1 above)
Comment (hub-bla) [reply client/package.json:15]: I think I did yarn command for installation and that's all

**Comment by hub-bla** [comment]
everything should be fine now

**Review by Safeheaded — APPROVED** [review]

**Pull request merged**